### PR TITLE
Custom QMK Firmware Keyboard

### PR DESCRIPTION
**Pull request by AidenRheem**
## Checklist before submitting proposal:

- [X] I am a current high school student (or younger).
- [X] I have submitted an RSVP at <https://hackclub.com/winter>
- [X] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [X] I have self-reviewed my project proposal.

### CODE REVIEW

File: AidenRheem.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Budget
+Materials
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Case/PCB (contains most materials I need) | [Bakeneko60](https://cannonkeys.com/products/bakeneko-60?variant=40911986720879) | $134.99 |
+| Switches        | [Gateron Oil King](https://divinikey.com/products/gateron-oil-king-linear-switches) | $54.35 |
+| Keycaps         | [Minimalistic BoW](https://idobao.net/collections/keycap/products/idobao-minimalist-black-and-white-pbt-keycaps-kit) | $46.50 |
+| Puller Tool     | [Keycap & Switch Puller](https://www.amazon.com/dp/B09L59DNH2) | $4.99 |
+| Shipping        | Divinikey (Switches)                  | $6.01  |
```
Comment (bellesea) [inline AidenRheem.md:22]: hii :) is this shipping or switches?

**Review by bellesea — CHANGES_REQUESTED** [review]
@AidenRheem thank you so much for sharing this!! What do you mean by custom QMK Firmware? are you making your custom keymap with QMK or are you writing your own version of QMK?

thank you :)

File: AidenRheem.md (hunk 1 above)
Comment (AidenRheem) [reply AidenRheem.md:22]: thats going to be shipping for the switches, sorry for the confusion

**Comment by AidenRheem** [comment]
> @AidenRheem thank you so much for sharing this!! What do you mean by custom QMK Firmware? are you making your custom keymap with QMK or are you writing your own version of QMK?
> 
> thank you :)

QMK is a software used to customize keyboards but I'm going to be writing my own version of QMK

**Comment by bellesea** [comment]
> > @AidenRheem thank you so much for sharing this!! What do you mean by custom QMK Firmware? are you making your custom keymap with QMK or are you writing your own version of QMK?
> > thank you :)
> 
> QMK is a software used to customize keyboards but I'm going to be writing my own version of QMK

Hey @AidenRheem ahhh cool!

The goal of Winter Hardware Wonderland is to make hardware hacking (creatively building things using the details of how electronics work) available to more people, but building a keyboard in the more mainstream sense (buying the keycaps, switches, PCB, case, then assembling them) doesn't really count. It just should be more lower level than plugging a few components together and uploading almost fully premade firmware (which is what QMK allows you to do?). Some customizations I might suggest: creating a custom layout, designing your own PCB, or compiling your own firmware.

For ideas, I'd highly recommend checking out the links in this Slack message: [hackclub.slack.com/archives/C0168BR5PDE/p1671582550645499](https://hackclub.slack.com/archives/C0168BR5PDE/p1671582550645499)

I'm going to close this pull request for now, but feel free to open a new one with a revised project proposal, or re-open this one if it's easier. I'm also dm-ing you on Slack and would be happy to continue chatting there :)

**Comment by AidenRheem** [comment]
@bellesea Updated my readme!

**Review by bellesea — APPROVED** [review]
@AidenRheem this is awesome! can't wait to see you build this and how you eventually approach building your custom firmware :) 

an awesome member of @hackclub/bank-ops will be reaching out soon with next steps!

**Review by hackliv — APPROVED** [review]
Hey @AidenRheem this looks awesome! I'd love to hear more about your plans for building this! I'm dming you on Slack now to set up some time to meet on a quick Zoom call!

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**